### DISCUSSION TITLE
Upscayl 2.11.0 Windows "Open With" Menu

### DISCUSSION THREAD

**Discussion by Suamy2KAr** [upscayl/upscayl#814] [Ideas]
Hola AMIGO!!. Saludos desde Argentina!! Consulto por el menu contextual sobre alguna imagen, en windows desde el explorador de archivos, al seleccionar "abrir con" y elegir Upscayl, no abre la imagen, debo abrirla manualmente. Sera que aun no esta implementado en tu programa o debo modificar alguna configuracion en el? Muchas Gracias!! Gran programa!!

Hi, friend!!. Greetings from Argentina!! I consult the context menu on an image, in Windows from the file explorer, when selecting "open with" and choosing Upscayl, it does not open the image, I must open it manually. Could it be that it is not yet implemented in your program or should I modify some configuration in it? Thank you so much!! Great program!!

**Comment by aaronliu0130** [comment]
Duplicate of #268.
> I have searched for this in the [issues tab](https://github.com/upscayl/upscayl/issues?q=).

![compulsory rickrolls are issued for duplicates of issues and the FAQ](https://user-images.githubusercontent.com/11874211/218282922-efab6795-84af-4cfb-9400-c28c8729ea78.gif)

**Reply by Suamy2KAr** [reply]
UPS!!! My bad... thnkz ;-)
